update: address the usage of 'acceptable_engine_tag' and correct typo (#12)
BerkeleyLab/nexport: README.md

              include_metadata=True, model_name="YOUR_PYTORCH_MODEL_NAME", model_author="YOUR_PYTORCH_MODEL_AUTHOR", activation_function="gelu",
              using_skip_connections=False)
```
`model`: **Required**. A PyTorch model instance.

> `acceptable_engine_tag`: **Required**. A string version number indicates the acceptable inference engine release. Since nexport generates the json file that is highly tied to the inference engine, the json keywords may subject to change whenever something happen on inference engine. 

Reviewer: ```suggestion
`acceptable_engine_tag`: **Required**. A string version number indicates an acceptable inference engine release. Because nexport generates a JSON file that is intimately tied to Inference-Engine, the JSON keywords may change when Inference-Engine's file format changes. 
```

> The is why this parameter is here to indicate the version track of inference engine.

Reviewer: ```suggestion
This parameter indicates an Inference-Engine `git` tag known to be able to read and write the nexport output.  Other Inference-Engine revisions might also be compatible.
```
